packages/mendel-config/src/bundle-config.js

```diff
@@ -7,6 +7,19 @@ function BundleConfig(id, options, config) {
     this.generator = options.generator || 'default';
 
     const {outdir} = config.baseConfig;
+    const supportedOptions = [
+        'entries',
+        'external',
+        'from',
+        'generator',
+        'manifest',
+        'outfile',
+        'outlet',
+        'require',
+        'runtime',
+    ];
```

Comment:
I think it would be better to infer from basic config. Actually, I thought the code already did that. I remember writing routes for picking only properties that existed on the default config from the parsed config. Maybe the code was removed. Either way, listing manually makes hard to explore and develop new features. Please get the list of supported options from `Object.keys(defaultConfig)`.